docs/PFC-test-plan.md

```diff
@@ -251,3 +255,26 @@ Verify Device Under Test (DUT) processes the PFC PAUSE frame with lossy prioriti
    * Keysight Rx port should receive all the 'Background data traffic' as well as 'Test data traffic'. There should not be any loss observed.
 5. Stop the PFC PAUSE storm.
 6. Repeat the test with a different lossless priority value (!=Pi).
+   
+### Test Case #3 - GLOBAL PAUSE (IEEE 802.3x port based Flow Control)
+
+<b>Note</b> : 802.3x Port based Flow Control is different than IEEE 802.1Qbb PFC (Priority Flow Control)
+#### Test Objective
+Verify Device Under Test (DUT) processes the GLOBAL PAUSE frame.
+#### Test Configuration
+
+- On SONiC Device Under Test (DUT) configure  lossless priority of all values Pi (0 \<= i \<=
+    7).
```

Comment:
We don't need to enable PFC on all the priorities. We just need to enable PFC on required priorities, e.g., 3 and 4.

Reply:
Taken care of